chain/network/src/routing.rs

```diff
@@ -338,7 +372,40 @@ impl RoutingTable {
             waiting_pong: SizedCache::with_size(PING_PONG_CACHE_SIZE),
             last_ping_nonce: SizedCache::with_size(PING_PONG_CACHE_SIZE),
             component_nonce,
+            peer_ibf_set: Default::default(),
+        }
+    }
+
+    pub fn exchange_routing_tables_using_ibf(
+        &self,
+        peer_id: &PeerId,
+        ibf_set: Rc<RefCell<IbfSet<SimpleEdge>>>,
+        ibf_level: ValidIBFLevel,
+        ibf_vec: &[IbfBox],
+        seed: u64,
+    ) -> (Vec<Edge>, Vec<u64>, u64) {
```

Comment:
Please add a comment on what the return value is. From its signature it is not obvious at all what is returned